### ISSUE TITLE
[SR-11855] https://lldb.llvm.org not up-to-date for Swift

### ISSUE THREAD

**Issue by dabrahams** [swiftlang/llvm-project#4434]
|                  |                 |
   |------------------|-----------------|
   |Previous ID       | SR-11855      |
   |Radar             | rdar://problem/57549483         |
   |Original Reporter | @dabrahams      |
   |Type              | Bug    |

   <details>
  <summary>Additional Detail from JIRA</summary>

   |                  |                 |
   |------------------|-----------------|
   |Votes             | 0         |
   |Component/s       | LLDB for Swift    |
   |Labels            | Bug        |
   |Assignee          | None      |
   |Priority          | Medium      |

   

   md5: 4636319bc4fe9fdfbe600a318a20b204

  </details>

**Issue Description:**

It says major benefits include

-   Up to date language support for C, C++, Objective-C

I figure if it doesn't mention Swift here, it's probably missing from many more importa nt places. Google finds about twice as many hits for ObjC as it does for Swift on that site ;-)

**Comment by beccadax** [comment]
@swift-ci create

**Comment by adrian-prantl** [comment]
That is intentional, because the version on llvm.org really doesn't support Swift. All the Swift support is in the swift-lldb fork on Github. Now that swift-lldb and LLVM share the same license we could discuss upstreaming the Swift support to llvm.org but that is also a tough sell because you cannot even build it without also patching LLVM because not all Swift support there has been upstreamed either. So in the short term we might want to add a sentence pointing to the existence of the fork and its relation to the version on llvm.org to the website.

**Comment by dabrahams** [comment]
Yeah, if I am looking to find out how to use LLDB from the command-line with Swift, LLVM.org is the most obvious place to look. That’s how I got here. So a pointer back to instructions for use from a terminal with Swift would be very helpful.

**Comment by MaxDesiatov** [comment]
When looking at <https://github.com/apple/swift-lldb,> apparently Swift fork of LLDB has been archived. Can we assume that changes have been upstreamed and this issue can be marked as resolved?

**Comment by keith** [comment]
I think this is still the case and it's just that everything has moved into apple/llvm-project as part of moving to the llvm monorepo.
